compiler/visq/visq

```diff
@@ -172,19 +242,48 @@ def _save_dot(circle_path: str, dot_path: str, metric: str, colors: list, qerror
     builder.save(qerror)
 
 
-def main():
-    # parse arguments
-    parser = _get_parser()
-    args = parser.parse_args()
-    _verify_args(args)
+def run_on_data_batchwise(fp32_model, q_model, data, dump_dot_graph, computers,
+                          batch_size):
 
-    fp32_model = args.fp32_circle
-    q_model = args.q_circle
-    data = None
-    if valid_attr(args, 'data'):
-        data = args.data
-    dump_dot_graph = args.dump_dot_graph
+    with tempfile.TemporaryDirectory() as model_dir:
+        fq_model = model_dir + '/fq_model.circle'
```

Comment:
is there a reason to fix this name?

Reply:
It stands for `fake_quantize_model`. As it's an internal it should be created in temporary directory to be removed as soon as possible.  IMHO there is no other reason to call it `fq_model` other than ease of use and for the full batch it's also called the same.

Reply:
OK, it is temporary file.

Reply:
maybe use `TemporaryFile()` not a fixed name?

Reply:
Yes, IMHO `TemporaryFile()` can also be used, but then IMHO we should also change it for `run_on_data`. IMHO this is not needful for this PR. May be could deal with later?

Reply:
OK got it